test/examples/break1.rb

```diff
@@ -4,6 +4,10 @@
 # A toy example for testing break commands.
 #
 class Break1Example
+  undef a if method_defined? :a
+  undef b if method_defined? :b
+  undef c! if method_defined? :c!
+
```

Comment:
I agree it would be nicer not to have those lines in the file. I'll try your suggestion and see if that works.